mono/mini/interp/transform.c

```diff
@@ -777,9 +777,30 @@ static int mono_class_get_magic_index (MonoClass *k)
 	return -1;
 }
 
+static void
+interp_generate_mae_throw (TransformData *td, MonoMethod *method, MonoMethod *target_method)
+{
+	MonoJitICallInfo *info = mono_find_jit_icall_by_name ("mono_throw_method_access");
+
+	/*
+	 * Inject code throwing MethodAccessException
+	 * Using MINT_LDFTN just to load constant on the stack.
+	 */
```

Comment:
@lewurm Any suggestion on doing this in a better way ?

Reply:
maybe use `MINT_MONO_LDPTR` for loading `method` and `target_method`; it does the same, but it makes it more clear that it is about arbitrary data instead of a function pointer

Reply:
@lewurm yes, that is what I was looking for, but missed it